app/services/discussion_service.rb

```diff
@@ -28,6 +28,7 @@ def self.create(discussion:, actor:)
     return false unless discussion.valid?
 
     discussion.save!
+    DiscussionReader.for(discussion: discussion, user: actor).set_volume! :loud
```

Comment:
It is a change in behaviour because email on participation used to be true by default, but this PR makes it false by default